### ISSUE TITLE
6 GB of GPU memory for HD inference?

### ISSUE THREAD

**Issue by altair199797** [hzwer/arXiv2021-RIFE#259]
Hello,
in your paper you state that you need 3.1 GB of GPU memory to process 1080p images, however when I run `model.inference(im0,im2)` it needs 4 GB of memory and the model alone needs around 2 GB of memory (so 6GB total). I am definetly not using your Large version, because the vimeo PSNR is 35.61, which is the PSNR of the normal (not large) model. Am I doing something wrong or have a misconception at some point?
Thank you in advance!

**Comment by hzwer** [comment]
I ran testtime.py at 1080p and got 9.9GB of memory overhead.
I found that after removing the following lines of code.
https://github.com/hzwer/arXiv2021-RIFE/blob/d6a21eac6b1eb23759519d11ec886c9fa033de2b/benchmark/testtime.py#L13-L15
The GPU memory overhead record is 2.5GB.

I'm guessing GPU memory overhead is also platform and even torch version dependent. Looking forward to your sharing of further experimental results.
